README.md

```diff
@@ -0,0 +1,183 @@
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[html5 page]**
+- **[css page]**
+- **[node modules]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps:
+
+- Create a new GitHub repo.
+- Create a new branch according to the GitHub flow rules.
+- Add a .gitignore file.
+- Add node_modules to gitignore file.
+- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+- Add descriptive README file to your project - please use this template but remember to customize it to your project.
+- Add 3 files (each one in a separate commit):
+
+	- A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
+	- A .css file that includes one class with styles for your header (make it in your favorite color).
+	- A test.md file with the text "This file should be ignored by git".
+
+- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
+Open a pull request.
+
+<!-- AUTHORS -->
+
+##: 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@allenkays](https://github.com/allenkays)
+- Twitter: [@allenkays](https://twitter.com/allenkays)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/allan-sang/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[linters for other stacks]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project can give me a star for this repo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Ajrass and May for their immense support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] It seems like the MIT links is not working properly and taht's because you don't have a license file in your directory, so please create a license file which you can [find here](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and attach the name of that file in place of the `./LICENSE` text you see at line `179`